src/common-model/functions.ts

```diff
@@ -172,23 +172,35 @@ export function getClinicalEntitySubmittedData(
   return clinicalRecords;
 }
 
+export const DonorCompletionFields: Array<keyof Donor | EntityAlias> = [
+  'completionStats',
+  'specimens',
+  'followUps',
+  'treatments',
+  'primaryDiagnoses',
+  ClinicalEntitySchemaNames.REGISTRATION,
```

Comment:
it feels odd to me to reference one value of a different enum here. particularly because these values are camelCased and `ClinicalEntitySchemaNames` are snake cased.

Reply:
Yes, I looked into how I can smooth this out yesterday, it's a side effect of sample registrations / specimen record core completion being intertwined, working on it

Reply:
OK I've updated this. `donorCompletionFields` now only includes fields found on Donor core completion. 

`extractEntityDataFromDonors` handling for Sample Registration is also updated.
This could probably be improved a little further for consistency, by moving this handling into `getClinicalEntitySubmittedData`, where all other entities are handled. I'll update this as part of #909

Reply:
sounds good thank you